packages/metro-config/src/defaults/__tests__/exclusionList-test.js

```diff
@@ -25,14 +25,84 @@ describe('exclusionList', () => {
     path.sep = originalSeparator;
   });
 
-  it('converts forward slashes in the RegExp to the OS specific path separator', () => {
+  it('proves we can write to path.sep for setting up the tests', () => {
     // $FlowFixMe: property sep is not writable.
     path.sep = '/';
+    expect(require('path').sep).toBe('/');
+    // $FlowFixMe: property sep is not writable.
+    path.sep = '\\';
+    expect(require('path').sep).toBe('\\');
+  });
+
+  it('converts forward slashes in the RegExp to the OS specific path separator in macOS/linux', () => {
+    // $FlowFixMe: property sep is not writable.
+    path.sep = '/';
+    // Simple case
     expect('a/b/c').toMatch(exclusionList([new RegExp('a/b/c')]));
+    expect('a/b/c').toMatch(exclusionList([/a\/b\/c/]));
```

Comment:
Would you mind adding a test for the case when the passed in path is a 'string'. The line 40 & 41 test the same path since both create a `RegExp`.

Reply:
I already have test cases cover that. See line 67-69